### ISSUE TITLE
Really exciting but I'm itchin for that youtube tutorial

### ISSUE THREAD

**Issue by funkmaster3000** [HolographicWings/LethalSDK-Unity-Project#1]
I've figured out a few things but there's not a lot of chatter at all yet about creating new maps. I can't figure out how to get any assets from the game to load alongside this stuff. I am a bonehead novice but this seems super cool

**Comment by subzerohurley** [comment]
Agreed, the Youtube tut would be awesome! Do you have a discord peeps can join for help too?

**Comment by broilerosu** [comment]
> I'll open a discord if the mod and sdk works well

you should open a thread in the actual modding discord for this game the one thunderstore links on their page https://discord.gg/XeyYqRdRGC

**Comment by AxelMGrey** [comment]
I can’t even get it to work. I made moons than doesn’t work.  Can you explain better for me?  I use Unity daily anwyays, so I know how Unity works.  Just the read me is confusing to me lol.

**Comment by AxelMGrey** [comment]
I just trying to make custom moons with the scene builder you provided.
Sent from my iPhone

On Dec 11, 2023, at 4:19 PM, DENIEL-VALERO Ewan ***@***.***> wrote:

﻿

I can’t even get it to work. I made moons than doesn’t work. Can you explain better for me? I use Unity daily anwyays, so I know how Unity works. Just the read me is confusing to me lol.

Make a video where you show every object of your mod in inspector, i will tell you if something is wrong
I still have a lot to do, fix or add before to make a video tutorial

**Comment by HolographicWings** [comment]
> I just trying to make custom moons with the scene builder you provided.

Yes, but i need an overall video of what you done
